### PR TITLE
feat: delete effect for jvm sdk

### PR DESCRIPTION
**Pull request by aludwiko**

### CODE REVIEW

**Review by johanandren — COMMENTED** [review]
Looks good, needs a proxy release+bump with the protocol change and then some docs in `docs/src/modules/java/pages/views.adoc`

**Review by octonato — APPROVED** [review]
Looks good. 

Left a small suggestion, nothing major.

**Review by johanandren — APPROVED** [review]
LGTM but don't merge until we have the proxy out or else we can't release other changes in the SDK until the proxy protocol change has reached production.

**Comment by johanandren** [comment]
Oh, also, docs/example still missing. Can be follow up, but make sure it happens (easy to forget)

**Comment by aludwiko** [comment]
documentation in separate task: https://github.com/lightbend/kalix-jvm-sdk/issues/1200

File: sdk/java-sdk/src/main/scala/kalix/javasdk/impl/view/ViewsImpl.scala
Comment (aludwiko) [reply sdk/java-sdk/src/main/scala/kalix/javasdk/impl/view/ViewsImpl.scala]: fixed

**Comment by octonato** [comment]
Merging since we have a separate issue for the docs.

**Comment by johanandren** [comment]
@octonato Note that now we can't release the SDK until the delete PR is merged and published (should be ready soon anyway but just FYI)

**Comment by johanandren** [comment]
Oh, the protocol change went in, but not the logic, so maybe it is fine. NVM

**Comment by octonato** [comment]
Oeps, I didn't pay attention to that. Sorry!

Well, why do think is fine? If we release it now, people will be able to use the API, but the proxy won't do anything.

**Comment by octonato** [comment]
Ah, I think I get your point now. As long it is not documented, we won't have the risk of a user relying on it. Or at least, the risk is minimal.

**Pull request merged**
